packages/preview/ntnu-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "ntnu-thesis"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Saim Iqbal"]
+repository = "https://github.com/saimnaveediqbal/thesis-NTNU-typst"
+license = "MIT"
+description = "An NTNU thesis template"
+categories = ["thesis"]
+keywords = ["thesis", "bachelor", "master", "ntnu", "university"]
```

Comment:
Examples should be added as `exclude = ["example.pdf"]` in order to not bloat the package size unnecessarily.